src/compiler/c.rs

```diff
@@ -884,7 +884,7 @@ fn remember_include_file(
 
     let original_path = path;
     // Canonicalize path for comparison; Clang uses ./header.h.
-    if path.starts_with(b"./") {
+    if path.starts_with(b"./") || path.starts_with(br".\") {
```

Comment:
In theory a ".\foo" file could literally exist on Unix, and that would break in that case.

Reply:
I'll special case this for Windows, thanks.